src/components/LoadingIndicator/LoadingIndicator.test.tsx

```diff
@@ -1,48 +1,112 @@
+/* eslint-disable no-unused-vars */
 import React from 'react';
 import { render } from '@testing-library/react';
-import { LoadingIndicator, LoadingIndicatorProps, TEST_ID } from './LoadingIndicator';
+import { LoadingIndicator, LoadingIndicatorProps } from './LoadingIndicator';
 
-const renderComponent = (props: LoadingIndicatorProps) => render(<LoadingIndicator {...props} />);
+enum MockSpinnerPositionType {
```

Comment:
Not sure why this was causing linting warning but likely I'm missing something

Reply:
'MockSpinnerPositionType' is defined but never used.

no unused vars warning. added /* eslint-disable no-unused-vars */. Will try figure out why thats being raised, shouldn't be